packages/genesys-spark-components/src/components/stable/gux-pagination/tests/gux-pagination.spec.ts

```diff
@@ -30,81 +30,120 @@ describe('gux-pagination', () => {
+      },
+      {
+        currentPage: 1,
+        totalItems: 1000,
+        itemsPerPage: 25,
+        layout: 'advanced',
+        disabled: true
       },
       {
         currentPage: 1,
         totalItems: 1000,
         itemsPerPage: 50,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
       {
         currentPage: 1,
         totalItems: 1000,
         itemsPerPage: 75,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
       {
         currentPage: 1,
         totalItems: 1000,
         itemsPerPage: 100,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
       {
         currentPage: 1,
         totalItems: 1000,
         itemsPerPage: 25,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
       {
         currentPage: 10,
         totalItems: 1000,
         itemsPerPage: 25,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
       {
         currentPage: 10,
         totalItems: 1000,
         itemsPerPage: 50,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
       {
         currentPage: 10,
         totalItems: 1000,
         itemsPerPage: 75,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
       {
         currentPage: 10,
         totalItems: 1000,
         itemsPerPage: 100,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
+      },
+      {
+        currentPage: 1,
+        totalItems: 1000,
+        itemsPerPage: 25,
+        layout: 'simple',
+        disabled: false
+      },
+      {
+        currentPage: 1,
+        totalItems: 1000,
+        itemsPerPage: 25,
+        layout: 'simple',
+        disabled: true
       },
-      { currentPage: 1, totalItems: 1000, itemsPerPage: 25, layout: 'simple' },
       {
         currentPage: -3,
         totalItems: 1000,
         itemsPerPage: 25,
-        layout: 'advanced'
+        layout: 'advanced',
+        disabled: false
       },
-      { currentPage: -3, totalItems: 0, itemsPerPage: 25, layout: 'advanced' }
-    ].forEach(({ currentPage, totalItems, itemsPerPage, layout }, index) => {
-      it(`should render as expected (${index + 1})`, async () => {
-        const html = `
+      {
+        currentPage: -3,
+        totalItems: 0,
+        itemsPerPage: 25,
+        layout: 'advanced',
+        disabled: false
+      }
+    ].forEach(
+      ({ currentPage, totalItems, itemsPerPage, layout, disabled }, index) => {
+        it(`should render as expected (${index + 1})`, async () => {
+          const html = `
           <gux-pagination
             current-page="${currentPage}"
             total-items="${totalItems}"
             items-per-page="${itemsPerPage}"
             layout="${layout}"
+            disabled="${disabled}"
```

Comment:
Could we have disabled as an optional prop with a default value of maybe `false` so we don't always have to pass it in as a prop? Or maybe it's required to always pass it in, but just wanted to check.